modules/asr/Cyberon/LICENSE.rst

```diff
@@ -0,0 +1,5 @@
+Copyright (c) 2023 Cyberon Corporation, All Rights Reserved
+
+The DSpotter™  engine included with this software has an expiring development license provided for end-customer use in a non-production environment. This is meant to be used only for internal evaluation of the suitability of the product for end-customer business needs.  
+
+For use in production designs, a commercial licence must be obtained from Cyberon Corporation.  The production firmware will only operate when used on an XMOS XU316-1024-QF60B-C24-CY device.
```

Comment:
sorry - I didn't realise we had more generic wording in other sections - for consistency, please replace `an XMOS XU316-1024-QF60B-C24-CY device.` with `a specially licenced XMOS device.`

Reply:
Text updated.